client/src/components/Review Widget/ReviewList.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import ReviewTile from './ReviewTile.jsx'
+import axios from 'axios'
+import './cssFiles/reviewSection.css'
+
+const ReviewList = (props) => {
+  const [reviews, setReviews] = useState([])
+
+  useEffect(() => {
+    getReviews()
+  }, [])
+
+  const getReviews = () => {
+    axios.get('http://localhost:3000/reviews', {
+      params: {
+        sort: 'newest',
+        product_id: props.product_id
+      }
+    })
+      .then((data) => {
+        setReviews(data.data.results)
+      })
+      .catch((error) => {
+        console.error(error)
+      })
+  }
+
+  const reviewListTiles = []
+  reviews.map((reviewData) => { return (reviewListTiles.push(<ReviewTile reviewData={reviewData} />)) })
```

Comment:
We might need a key prop here as well for ReviewTitle. Again, the key prop is optional but it is for optimization. Maybe just something to keep in mind but these do cause a lot of console clutter.

Reply:
agreed, i'll work on it in the new branch